src/components/ContentSidebar/ActivityFeed/activity-feed/ActivityFeed.js

```diff
@@ -488,7 +500,7 @@ class ActivityFeed extends React.Component<Props, State> {
                         <EmptyState isLoading={isLoading} showCommentMessage={showApprovalCommentForm} />
                     ) : (
                         <ActiveState
-                            inlineError={activityFeedError}
+                            inlineError={activityFeedError ? this.createActivityFeedApiError() : undefined}
```

Comment:
Can you put all this into ```createActivityFeedApiError``` so the ternary isn't needed? Then create a basic UT for it for code coverage :)